### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by tarak-psdu**
This is the personal Portfolio: setup and mobile first design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="flex-container">
+           <h1 class="flex-item">Welcome</h1>
```
Comment (tugsem) [inline index.html]: - [x] Kindly use `a` tag for the logo for three reasons: convention, semantics and user experience.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  background-size: cover;
+  width: 100%;
+  margin: auto;
+}
+
+.flex-container {
+  margin: 20px;
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (tugsem) [inline style.css]: - [x] In order to match the given design please add a white background to navbar and align items also vertically. 

Here's what you need to build:
image

Here's what you build: 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+           <a href="#"><img class="flex-item" src="./images/Screenshot_1.jpg" alt=""></a>
+        </nav>
+    </header>
+    <section>
+        <h2>Hey there. I’m Mariane</br>
```
Comment (tugsem) [inline index.html]: - [x] This title should be tagged with `h1`. Kindly fix it. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        <div>
+            <img src="./images/medium.png" alt="Medium">
+        </div>
+        <div>
+            <img src="./images/github.png" alt="Github">
+        </div>
+        <div>
+            <img src="./images/angellist.png" alt="Angellist">
+        </div>
+    </section>
```
Comment (tugsem) [inline index.html]: - [x] The list of icons shoul be applied using a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Kindly  arrange it. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hi @tarak-psdu , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️Linters setup
✔️Great commit messages

## Required Changes ♻️
- [x]  Please provide a short summary in your pull request description.
 - [x]  Please add readme template provided by microverse(https://github.com/microverseinc/readme-template/blob/master/README.md) and provide all main headers filled with your information.
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tarak-psdu) [reply index.html]: I added the `a` tag to the logo section.

File: style.css (hunk 2 above)
Comment (tarak-psdu) [reply style.css]: Fixed

File: index.html (hunk 3 above)
Comment (tarak-psdu) [reply index.html]: Fixed

File: index.html (hunk 4 above)
Comment (tarak-psdu) [reply index.html]: Fixed

**Review by frank1738 — APPROVED** [review]
Hi @tarak-psdu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**